<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogConfigModule</class>
 <widget class="QDialog" name="DialogConfigModule">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>141</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Module type :</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>231</width>
     <height>38</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Git</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Bzr</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Patch</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Archive source</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Single file</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Inline script</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Shell command</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>141</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>URL :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="textURL">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>60</y>
     <width>231</width>
     <height>38</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>141</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Version :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="textVersion">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>110</y>
     <width>231</width>
     <height>38</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>141</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Command :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="textCommand">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>160</y>
     <width>231</width>
     <height>38</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogConfigModule</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogConfigModule</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
